index.html

```diff
@@ -15,76 +15,89 @@
             <h2 class="work">I am a software developer</h2>
             <p class="service">
                 I can help you build a product , feature or website Look through some 
                 of my work and experience! If you like what you see and have a project 
                 you need coded, don’t hestiate to contact me.
             </p>
-        </div>
-        <!-- end of welcome message -->
-        <!-- social media -->
-        <ul class="social_media">
-            <li>
-                <a href="https://github.com/Victor-chiemerie">
-                    <img src="images/github.png" alt="github">
-                </a>
-            </li>
-            <li>
-                <a href="https://www.linkedin.com/in/victor-chiemerie-460475246/">
-                    <img src="images/linkedin.png" alt="linkedin">
-                </a>
-            </li>
-            <li>
-                <a href="https://twitter.com/VictorChiemer20">
-                    <img src="images/Twitter.png" alt="twitter">
-                </a>
-            </li>
-            <li>
-                <a href="#">
-                    <img src="images/medium.png" alt="medium">
-                </a>
-            </li>
-            <li>
-                <a href="#">
-                    <img src="images/angel list.png" alt="angellist">
-                </a>
-            </li>
-        </ul>
-        <!-- end of social media -->
+          </div>
+          <!-- end of welcome message -->
+          <!-- social media -->
+          <ul class="social_media">
+              <li>
+                  <a href="https://github.com/Victor-chiemerie">
+                      <img src="images/github.png" alt="github">
+                  </a>
+              </li>
+              <li>
+                  <a href="https://www.linkedin.com/in/victor-chiemerie-460475246/">
+                      <img src="images/linkedin.png" alt="linkedin">
+                  </a>
+              </li>
+              <li>
+                  <a href="https://twitter.com/VictorChiemer20">
+                      <img src="images/Twitter.png" alt="twitter">
+                  </a>
+              </li>
+              <li>
+                  <a href="#">
+                      <img src="images/medium.png" alt="medium">
+                  </a>
+              </li>
+              <li>
+                  <a href="#">
+                      <img src="images/angel list.png" alt="angellist">
+                  </a>
+              </li>
+          </ul>
+          <!-- end of social media -->
+         </div>
     </section>
     <!-- work pages -->
     <section class="work-pages">
       <main class="first-page">
-        <div>
-          <h2 class="title"> My Recent Works </h2>
+        <div class="work-container">
+          <div class="work-item1">
+            <h2 class="title"> My Recent Works </h2>
             <img class="line" src="images/line.png" alt="line">
```

Comment:
Please as seen in the figma template, I recommend that you try making your version more like it.
Currently, the text part is covering almost the whole screen. Kindly fix this to make the UI more attractive.


**Your design**

<img width="908" alt="fix" src="https://user-images.githubusercontent.com/84930355/206862425-e10837ec-5c36-4205-b9f1-617495e7c91b.png">

Also, I recommend fixing the alignment of the text inside of the cards in the image below.


<img width="578" alt="looom" src="https://user-images.githubusercontent.com/84930355/206862639-489168c6-346e-4613-b920-899838821ef9.png">



**Figma design**

<img width="284" alt="desktoooooop" src="https://user-images.githubusercontent.com/84930355/206862469-5d4a0900-0aba-46b5-b3e8-c8ba84e119ae.png">